py/src/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.py

```diff
@@ -61,6 +60,12 @@ class H2ODeepLearningParams(H2OAlgoSupervisedParams, HasStoppingCriteria, HasQua
         "Force reproducibility on small data (will be slow - only uses 1 thread)",
         H2OTypeConverters.toBoolean())
 
+    activation = Param(
+        Params._dummy(),
+        "activation",
+        "The activation function (non-linearity) to be used the neurons in the hidden layers.",
```

Comment:
* -> `The activation function (non-linearity) applied on neurons of hidden layers.`?